dom.bs

```diff
@@ -1195,31 +1190,35 @@ method, when invoked, must run these steps:
  <var>once</var>.
 </ol>
 
-<p>To <dfn>remove an event listener</dfn>, given an {{EventTarget}} object <var>eventTarget</var>
-and an <a>event listener</a> <var>listener</var>, set <var>listener</var>'s
-<a for="event listener">removed</a> to true and <a for=list>remove</a> <var>listener</var> from
-<var>eventTarget</var>'s <a for=EventTarget>event listener list</a>.
-<!-- Intentionally not exported. -->
+<p>To <dfn export>remove an event listener</dfn>, given an {{EventTarget}} object
+<var>eventTarget</var> and an <a>event listener</a> <var>listener</var>, run these steps:
+
+<ol>
+ <li><p>If the <a>context object</a> is a {{ServiceWorkerGlobalScope}} object and its
+ <a for="ServiceWorkerGlobalScope">service worker</a>'s
+ <a for="service worker">set of event types to handle</a> contains <var>type</var>, then
+ <a>report a warning to the console</a> that this might not give the expected results.
+ [[!SERVICE-WORKERS]]
```

Comment:
To give the context of what the expected results would be, does leaving a short note along the lines of "Service Workers will still be started after the removal." make sense?